### PR TITLE
skipped pics lose 1 karma + you can get karma of a pic

### PR DESCRIPTION
**Pull request by BoostedBoat**
!!! Don't merge while PR 90 has not been merged, in need of refractoring !!!

### CODE REVIEW

**Comment by leonardvaney** [comment]
Can you modifiy the xml file of GuessPreviewActivity to remove the cirrus errors?

**Review by JeremyDezalos — APPROVED** [review]
Remove the commented function otherwise it's fine

File: app/src/androidTest/java/com/github/wnder/GuessPreviewActivityTest.java [hunk 1]
```diff
@@ -56,6 +58,24 @@ public void testGuessLocationButton(){
...
+            onView(withId(R.id.skipButton)).perform(click());
+            Intents.intended(hasComponent(GuessPreviewActivity.class.getName()));
+            pic.onKarmaAvailable((k2) -> {
+                assertThat(k2, is(k1-1));
+            });
+
+
+        });
+    }*/
+
```
[the remark this answers is not in the document]
Comment (BoostedBoat) [reply app/src/androidTest/java/com/github/wnder/GuessPreviewActivityTest.java]: Deleted

File: app/src/main/java/com/github/wnder/GuessPreviewActivity.java [hunk 2]
```diff
@@ -14,6 +14,11 @@
 import android.widget.Button;
 import android.widget.ImageView;
 
+import com.github.wnder.picture.ExistingPicture;
```
Comment (stblr) [inline app/src/main/java/com/github/wnder/GuessPreviewActivity.java:17]: Duplicate

File: app/src/main/java/com/github/wnder/GuessPreviewActivity.java [hunk 3]
```diff
@@ -14,6 +14,11 @@
...
+
+import java.util.concurrent.ExecutionException;
+import java.util.function.Consumer;
+
 import androidx.appcompat.app.AppCompatActivity;
```
Comment (stblr) [inline app/src/main/java/com/github/wnder/GuessPreviewActivity.java:22]: Duplicate

**Review by stblr — CHANGES_REQUESTED** [review]

**Review by stblr — APPROVED** [review]

**Pull request merged**